ruled_labels/[DRAFT]specs_monorepo.yaml

```
@@ -0,0 +1,187 @@
+    color: "5319e7"
+  - name: F3-breaks_RPC_API
+    description: This PR changes the RPC API.
+    color: cdadad
+  - name: F4-breaks_runtime_API
+    description: This PR changes the runtime API.
+    color: cdadad
+  - name: F4-breaks_host_function
+    description: This PR changes a host function.
+    color: cdadad
+  - name: I0-panic
+    description: The node panics and exits without proper error handling.
+    color: 17b25d
+  - name: I1-security
+    description: The node fails to follow expected, security-sensitive, behaviour.
+    color: d93f0b
+  - name: I2-bug
+    description: The node fails to follow expected behavior.
+    color: "249899"
+  - name: I3-annoyance
+    description: The node behaves within expectations, however this “expected behaviour” itself is at issue.
+    color: D6075D
+  - name: I4-refactor
+    description: Code needs refactoring.
+    color: 0075ca
+  - name: I5-enhancement
+    description: An additional feature request.
+    color: c2e0c6
+  - name: I6-meta
+    description: A specific issue for grouping tasks or bugs of a specific category.
+    color: a2eeef
+  - name: I7-duplicate
+    description: Issue is a duplicate. Closer should comment with a link to the duplicate.
+    color: D62209
+  - name: I8-wont_fix
+    description: Issue is in principle valid, but this project will not address it. Closer should explain why.
+    color: FB3701
+  - name: I9-invalid
+    description: Issue is invalid. Closer should comment why.
+    color: F76E02
+  - name: I10-optimisation
+    description: An enhancement to provide better overall performance in terms of time-to-completion for a task.
+    color: c5def5
+  - name: T0-node
+    description: This PR/Issue is related to the topic “node”.
+    color: fbffe0
+  - name: T1-runtime
+    description: This PR/Issue is related to the topic “runtime”.
+    color: F5FCE6
+  - name: T2-RPC_API
+    description: This PR/Issue is related to RPC APIs.
+    color: 009B40
+  - name: T3-runtime_API
+    description: This PR/Issue is related to runtime APIs.
+    color: 009B40
+  - name: T4-host_functions
+    description: This PR/Issue is related to host functions.
+    color: 009B40
+  - name: T5-XCM
+    description: This PR/Issue is related to XCM.
+    color: 006b75
+  - name: T6-smart_contracts
+    description: This PR/Issue is related to smart contracts.
+    color: 0C7BAD
+  - name: T7-parachains_engineering
+    description: This PR/Issue is related to Parachains performance, stability, maintenance.
+    color: 0052cc
+  - name: T8-parachains_protocol
+    description: This PR/Issue is related to Parachains features and protocol changes.
+    color: 5319e7
+  - name: T9-polkadot
+    description: This PR/Issue is related to/affects Polkadot.
+    color: ffeeee
+  - name: T10-tests
+    description: This PR/Issue is related to tests.
+    color: 6C9C98
+  - name: T11-documentation
+    description: This PR/Issue is related to documentation.
+    color: f48a75
```

```suggestion
  - name: T11-documentation
    description: This PR/Issue is related to documentation.
    color: f48a75
  - name: T12-benchmarks
    description: This PR/Issue is related to benchmarking and weights.
    color: ???
```

Dont know what color to pick.